src/renderer/wg_engine/tvgWgCommon.cpp

```diff
@@ -27,18 +27,22 @@
 // context
 //*****************************************************************************
 
-void WgContext::initialize()
+void WgContext::initialize(void* disp_inst, void* wind_serf, uint32_t w, uint32_t h)
```

Comment:
please take care of the coding convention. thorvg doesn't use `a_b` 
disp_inst -> display
wind_serf -> window